lms/models/event.py

```diff
@@ -15,6 +15,8 @@ class Type(str, Enum):
         DEEP_LINKING = "deep_linking"
         AUDIT_TRAIL = "audit"
         EDITED_ASSIGNMENT = "edited_assignment"
+        SCORED = "scored"
```

Comment:
New event types? Are this good names? 

`STUDENT_SCORED`? `STUDENT_SUBMISSION`?